docs/tutorial.rst

```diff
@@ -1025,16 +1018,86 @@ We'll count up the number of neutral and deleterious mutations in the three regi
             f"and {d} deleterious mutations."
         )
 
-    # From 0 to 1000: 3 neutral mutations and 0 deleterious mutations.
-    # From 1000 to 2000: 1 neutral mutations and 2 deleterious mutations.
-    # From 2000 to 3000: 1 neutral mutations and 0 deleterious mutations.
+    # From 0 to 1000: 50 neutral mutations and 0 deleterious mutations.
+    # From 1000 to 2000: 12 neutral mutations and 20 deleterious mutations.
+    # From 2000 to 3000: 45 neutral mutations and 0 deleterious mutations.
 
 This verifies that there are only deleterious mutations in the center bit.
```

Comment:
```suggestion
This verifies that there are only deleterious mutations in the center bit (as expected under the Gamma_K17).
```

Reply:
I think my wording here was actually very confusing. What we actually mean is: "This verifies that deleterious mutations can only be found in the center bit, given we applied the Gamma_K17 DFE just to this region."

Right now it can read like only deleterious mutations happen in the center bit, I think.